WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -559,10 +564,11 @@ def __doOSG ( self ):
     """ Treat the OSG case """
 
     osgDir = ''
+    vo = self.pp.releaseProject.replace( 'DIRAC', '' ).upper()
```

Comment:
I just removed the whole doOSG function. My pilots worked without creating another subfolder and all the workingdirectories are created by the batch system as you say.